src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParser.cs

```diff
@@ -988,7 +979,7 @@ internal void ScanBlank()
          * Scans chars following a '\' (not counting the '\'), and returns
          * a RegexNode for the type of atom scanned.
          */
-        internal RegexNode ScanBackslash()
+        internal RegexNode ScanBackslash(bool scanOnly = false)
```

Comment:
IMO, optional parameters for these internal APIs aren't a great idea.  By being non-optional it ensures there isn't a caller missed.